src-backbone/app/js/models/session.js

```diff
@@ -0,0 +1,107 @@
+var AUTH_TOKEN_KEY = 'auth_token';
+var STORAGE_KEY = AUTH_TOKEN_KEY;
+
+var Config = require('utils/config');
+var Helpers = require('utils/helpers');
+
+var SessionModel = Backbone.Model.extend({
+
+    initialize: function() {
+        var me = this;
+        $.ajaxPrefilter(function(options, originalOptions, jqXHR) {
+            options.url = Config.API_BASE + options.url;
+            if (typeof me.get(AUTH_TOKEN_KEY) !== 'undefined') {
```

Comment:
Is there a cleaner way of doing this check? I'm guessing `if (me.get(AUTH_TOKEN_KEY))` doesn't work?